Warning, the following versions were found for plone.portlet.collection
3.0.3
3.0.2
3.0.1
3.0
2.2.1
2.2.0
2.1.7
2.1.6
2.1.5
2.1.4
2.1.3
2.1.2
2.1.1
2.1
2.0.4
2.0.3
2.0.2
2.0.1
2.0
1.1.3
1.1.2
1.1.0
1.1rc1
1.0b1
0.1.1
0.1
Using 3.0.3
Use --version to specify a different version.
Using url https://pypi.python.org/packages/source/p/plone.portlet.collection/plone.portlet.collection-3.0.3.tar.gz (21 KB) for plone.portlet.collection.
Downloading plone.portlet.collection
Unpacking plone.portlet.collection...
done
working in /tmp/tmp8TEUGiconda_skeleton_plone.portlet.collection-3.0.3.tar.gz
Fetching package metadata: ......

WARNING: conda-build appears to be out of date. You have version 1.13.0 but the
latest version is 1.14.1. Run

conda update -n root conda-build

to get the latest version.

Solving package specifications: .patching file core.py
Hunk #1 succeeded at 168 with fuzz 2 (offset 2 lines).

WARNING: conda-build appears to be out of date. You have version 1.13.0 but the
latest version is 1.14.1. Run

conda update -n root conda-build

to get the latest version.


The following NEW packages will be INSTALLED:

    numpy:      1.9.2-py27_0 
    openssl:    1.0.1k-1     
    pip:        7.0.3-py27_0 
    python:     2.7.10-0     
    pyyaml:     3.11-py27_1  
    readline:   6.2-2        
    setuptools: 18.0.1-py27_0
    sqlite:     3.8.4.1-1    
    system:     5.8-2        
    tk:         8.5.18-0     
    yaml:       0.1.6-0      
    zlib:       1.2.8-0      

Linking packages ...
[                    ]|                                                  |   0%[openssl             ]|                                                  |   0%[readline            ]|####                                              |   8%[sqlite              ]|########                                          |  16%[system              ]|############                                      |  25%[tk                  ]|################                                  |  33%[yaml                ]|####################                              |  41%[zlib                ]|#########################                         |  50%[numpy               ]|#############################                     |  58%[python              ]|#################################                 |  66%[pyyaml              ]|#####################################             |  75%[setuptools          ]|#########################################         |  83%[pip                 ]|#############################################     |  91%[      COMPLETE      ]|##################################################| 100%
Applying patch: '/tmp/tmp8TEUGiconda_skeleton_plone.portlet.collection-3.0.3.tar.gz/pypi-distutils.patch'
Warning, the following versions were found for plone.app.contenttypes
1.2b2
1.2b1
1.2a9
1.2a8
1.2a7
1.2a6
1.2a5
1.2a4
1.2a3
1.2a2
1.2a1
1.1b5
1.1b4
1.1b3
1.1b2
1.1b1
1.1a1
1.0
1.0rc1
1.0b2
1.0b1
Using 1.2b2
Use --version to specify a different version.
Using url https://pypi.python.org/packages/source/p/plone.app.contenttypes/plone.app.contenttypes-1.2b2.tar.gz (197 KB) for plone.app.contenttypes.
Downloading plone.app.contenttypes
plone.app.cont   0% |                              | ETA:  --:--:--   0.00  B/splone.app.cont   8% |##                             | ETA:  0:00:00  33.72 MB/splone.app.cont  16% |#####                          | ETA:  0:00:00   2.55 MB/splone.app.cont  24% |#######                        | ETA:  0:00:00   3.75 MB/splone.app.cont  32% |##########                     | ETA:  0:00:00   4.69 MB/splone.app.cont  40% |############                   | ETA:  0:00:00   3.15 MB/splone.app.cont  48% |###############                | ETA:  0:00:00   3.68 MB/splone.app.cont  56% |#################              | ETA:  0:00:00   4.23 MB/splone.app.cont  65% |####################           | ETA:  0:00:00   4.76 MB/splone.app.cont  73% |######################         | ETA:  0:00:00   5.29 MB/splone.app.cont  81% |#########################      | ETA:  0:00:00   5.83 MB/splone.app.cont  89% |###########################    | ETA:  0:00:00   4.56 MB/splone.app.cont  97% |############################## | ETA:  0:00:00   4.94 MB/splone.app.cont 100% |###############################| ETA:  0:00:00   5.04 MB/splone.app.cont 100% |###############################| Time: 0:00:00   5.00 MB/s
Unpacking plone.app.contenttypes...
done
working in /tmp/tmpo3RNxGconda_skeleton_plone.app.contenttypes-1.2b2.tar.gz
Solving package specifications: .patching file core.py
Hunk #1 succeeded at 168 with fuzz 2 (offset 2 lines).

Applying patch: '/tmp/tmpo3RNxGconda_skeleton_plone.app.contenttypes-1.2b2.tar.gz/pypi-distutils.patch'
WARNING: Could not add entry points. They were:

      # -*- Entry points: -*-
      [z3c.autoinclude.plugin]
      target = plone
      
Error: Could not parse: plone.namedfile [blobs]
